openshift/machine-config-operator
# [OCPNODE-852] workerlatency profiles - updating the kubelet configuration based on the nodes.config.openshift.io resource

**- What I did**
Added support to update the kubelet configuration based on the node.config.openshift.io spec changes

**-How to verify it**
Create/Update the nodes.config.openshift.io CR with the relevant/valid workerlatency profile and expect the corresponding changes in the Kubelet Configuration that are rolled out on all the worker nodes by MCO leveraging the KubeletConfig CR.

**- Description for the changelog**
Based on the [enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/worker-latency-profile/worker-latency-profile.md), users can create/modify the node.config.openshift.io resource which results in the corresponding  kubelet configuration changes on all the worker nodes.
